tests/test_backend.py

```diff
@@ -516,9 +515,6 @@ def foo(run):
     with h5py.File(db_dir / "extracted_data" / "p1234_r42.h5") as f:
         assert f["foo/data"][()] == 42
 
-    # Make sure that the GUI evaluates the context file correctly (which it does
-    # upon opening a database directory).
-    win = MainWindow(db_dir, False)
```

Comment:
We still need to test that the editor evaluates the context file correctly :stuck_out_tongue:

Reply:
Good point, I had forgotten that loading it in the editor also validated it in the target Python.